integration-tests/test_proxy.py

```diff
@@ -2,21 +2,20 @@
 import os
 import shutil
 import ssl
-from subprocess import check_call
 import time
+from subprocess import check_call
 
-import toml
-from tornado.httpclient import HTTPClient, HTTPRequest, HTTPClientError
 import pytest
-
-from tljh.config import (
-    reload_component,
-    set_config_value,
-    unset_config_value,
-    CONFIG_FILE,
-    CONFIG_DIR,
-    STATE_DIR,
-)
+import toml
+from tornado.httpclient import HTTPClient
+from tornado.httpclient import HTTPClientError
+from tornado.httpclient import HTTPRequest
+
+from tljh.config import CONFIG_DIR
```

Comment:
It is coming from reorder-python-imports by asolittle, similar to isort: https://github.com/asottile/reorder_python_imports#readme

It has been used in jupyterhub/jupyterhub, I've grown to appreciate it. Mainly because its groups the stdlib imports, third party imports, and local imports separately. The fact that it makes all imports into separate lines helps a bit with readability in my mind, but I don't care about that difference much.

I've rebased the commit applying changes from it now!